pipeline/differ/src/main/java/org/datacommons/pipeline/differ/DifferUtils.java

```diff
@@ -20,28 +30,59 @@ enum Diff {
     UNMODIFIED;
   }
 
+  public static final TupleTag<KV<String, String>> OBSERVATION_NODES_TAG =
+      new TupleTag<KV<String, String>>() {};
+  public static final TupleTag<KV<String, String>> SCHEMA_NODES_TAG =
+      new TupleTag<KV<String, String>>() {};
+
+  public static final String[] GRUOPBY_PROPERTIES = {
+    "variableMeasured",
+    "observationAbout",
+    "observationDate",
+    "observationPeriod",
+    "measurementMethod",
+    "unit",
+    "scalingFactor"
+  };
+
   /**
-   * Converts an MCF graph (SVObs only) to a PCollection of Key-Val pairs where key contains PVs
-   * excluding value.
+   * Converts an MCF graph to a PCollectionTuple of observation and schema nodes.
    *
    * @param graph input graph to process
-   * @return PCollection of Key-Val pairs
+   * @return PCollectionTuple of observation and schema nodes
    */
-  static PCollection<KV<String, String>> processGraph(PCollection<McfGraph> graph) {
-    PCollection<KV<String, String>> nodes =
-        graph.apply(
-            "ProcessGraph",
-            ParDo.of(
+  static PCollectionTuple processGraph(PCollection<McfGraph> graph) {
+    return graph.apply(
+        "ProcessGraph",
+        ParDo.of(
                 new DoFn<McfGraph, KV<String, String>>() {
                   @ProcessElement
                   public void process(ProcessContext c) {
                     McfGraph g = c.element();
-                    for (String[] pv : GraphUtils.getSeriesValues(g)) {
-                      c.output(KV.of(pv[0], pv[1]));
+                    for (Map.Entry<String, PropertyValues> entry : g.getNodesMap().entrySet()) {
+                      PropertyValues pvs = entry.getValue();
+                      Map<String, McfGraph.Values> pv = pvs.getPvsMap();
+                      if (org.datacommons.util.GraphUtils.isObservation(pvs)) {
+                        String key =
+                            Arrays.asList(GRUOPBY_PROPERTIES).stream()
+                                .map(prop -> GraphUtils.getPropertyValue(pv, prop))
+                                .collect(Collectors.joining(","));
+                        String value = GraphUtils.getPropertyValue(pv, Property.value.name());
+                        c.output(KV.of(key, value));
+                      } else {
+                        List<String> keys = new ArrayList<>(pv.keySet());
+                        Collections.sort(keys);
+                        String value =
+                            keys.stream()
+                                .map(key -> GraphUtils.getPropertyValue(pv, key))
+                                .collect(Collectors.joining(","));
```

Comment:
Since this will be written as a csv with a header with only 4 fields, it might be confusing if these also use commas to separate (and might be hard to parse the csv) - is there a different way to display the values? 

Also would it be helpful to include the properties too? (For example if the property names change but the values stay the same, or if a node adds/deletes pvs, so it's easy to match which values correspond)